src/components/shared/layout/toggleHeader.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div class="collapsable-header" @click="_collapsed = !_collapsed">
+        <div class="title">{{ title }}</div>
+        <div class="caret">
+            <croud-dropdown-indicator-button :collapsed="_collapsed"/>
+        </div>
+    </div>
+</template>
+
+<script>
+    import CroudDropdownIndicatorButton from '../buttons/DropdownIndicatorButton'
+
+    /**
+    * A basic toggle header showing a collapsed or expanded state
+    *
+    * @example ./croud-toggle-header.md
+    */
+    export default {
+        name: 'croud-toggle-header',
+
+        components: {
+            CroudDropdownIndicatorButton,
+        },
+
+        props: {
+            /**
+             * Set the title of the header
+             */
+            title: {
+                type: String,
+                default: 'Title',
+            },
+
+            /**
+             * Collapsed prop - emits update:collapsed event compatible with prop.sync
+             */
+            collapsed: {
+                type: Boolean,
+                default: false,
+            },
+        },
+
+        computed: {
+            _collapsed: {
+                get() {
+                    return this.collapsed
+                },
+                set(val) {
+                    this.$emit('update:collapsed', val)
+                },
+            },
+        },
+    }
+</script>
+
+<style lang="scss" scoped>
+    @import '../../../resources/sass/variables/_all.scss';
+
+    $border-size: 1px;
+    $border-colour: $croud-colour-grey-lighter;
+
+    .collapsable-header {
+        display: flex;
+        align-items: center;
+        height: 35px;
+        width: 100%;
+        border: solid $border-size $border-colour;
+        padding: 0 10px;
+
+        font-family: $croud-font-body;
```

Comment:
Wouldn't these rules be inherited?

Reply:
I have removed the rules that should be inherited, i have left the ones which i feel should be specified.